.github/pr-labeler.config.yml

```diff
@@ -1,13 +1,50 @@
 e2e:
   - any:
       - changed-files:
-          - any-glob-to-any-file: "packages/wrangler/e2e/**/*"
+          - any-glob-to-any-file:
+              [
+                "packages/wrangler/**/*",
+                "packages/miniflare/**/*",
+                "packages/vite-plugin-cloudflare/**/*",
+                "packages/create-cloudflare/**/*",
+                "packages/vitest-pool-workers/**/*",
+              ]
       - head-branch: "changeset-release/main"
 
 c3-e2e:
   - any:
       - changed-files:
-          - any-glob-to-any-file: "packages/create-cloudflare/e2e-tests/**/*"
-          - any-glob-to-any-file: "packages/create-cloudflare/templates/**/*"
-          - any-glob-to-any-file: "packages/create-cloudflare/templates-experimental/**/*"
+          - any-glob-to-any-file:
+              [
+                "packages/wrangler/**/*",
+                "packages/miniflare/**/*",
+                "packages/vite-plugin-cloudflare/**/*",
+                "packages/create-cloudflare/**/*",
+                "packages/vitest-pool-workers/**/*",
+              ]
```

Comment:
These seem overzealous to me... Why do C3 E2Es need to run on vitest pool workers changes, for instance?